### ISSUE TITLE
[Enhancement]: Universal/Inter Library Search

### ISSUE THREAD

**Issue by jt196** [advplyr/audiobookshelf#556]
### Describe the feature/enhancement

So, have a bunch of radio shows and audiobooks. Some of them have an author that exists in both libraries - e.g. Agatha Christie. When I search in Plex, I get a list of all the items in both libraries. Admittedly, there's 2 versions of the author for each library, but it's a useful shortcut.

**Comment by cyberkoala** [comment]
This might also make opening up multiple libraries more manageable. 
Use cases:
Large multi user deployments may make libraries get unwieldy, if you have 200 or so podcasts, half of which are still updating. 
Separation of categories - library for news podcasts, library for entertainment. 
Future proofing if the experimental ebook reader functionality becomes more fully fleshed, meaning most deployments might have 3 different standard libraries..

**Comment by adocampo** [comment]
Yes, I have the podcast and three libraries, I would like the search tool to search in all my libraries and podcasts, and not to be forced to search in each library for some term.
